File: sdk/java/src/main/java/com/aiot/sdk/AiotClient.java
package com.aiot.sdk;

public class AiotClient {
    private String endpoint;

    public AiotClient(String endpoint) {
        this.endpoint = endpoint;
    }

    public boolean ping() {
        return true;
    }

    public boolean start() {
        return true;
    }

    public boolean stop() {
        return true;
    }
}

File: sdk/csharp/AiotClient.cs
namespace Aiot.Sdk
{
    public class AiotClient
    {
        private string _endpoint;

        public AiotClient(string endpoint)
        {
            _endpoint = endpoint;
        }

        public bool Ping()
        {
            return true;
        }

        public bool Start()
        {
            return true;
        }

        public bool Stop()
        {
            return true;
        }
    }
}
